dotnet/samples/AgentSyntaxExamples/Example02_Plugins.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using AgentSyntaxExamples;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Agents;
+using Microsoft.SemanticKernel.Agents.Extensions;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.OpenAI;
+using Plugins;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Examples;
+
+/// <summary>
+/// Demonstrate creation of <see cref="ChatCompletionAgent"/> with a <see cref="KernelPlugin"/>,
+/// and then eliciting its response to explicit user messages.
+/// </summary>
+public class Example02_Plugins : BaseTest
+{
+    [Fact]
+    public async Task RunAsync()
+    {
+        // Define the agent
+        ChatCompletionAgent agent =
+            new(
+                kernel: this.CreateKernelWithChatCompletion(),
+                instructions: AgentInventory.HostInstructions,
+                name: AgentInventory.HostName)
+            {
+                ExecutionSettings = new OpenAIPromptExecutionSettings() { ToolCallBehavior = ToolCallBehavior.AutoInvokeKernelFunctions }
+            };
+
+        // Initialize plugin and add to the agent's Kernel (same as direct Kernel usage).
+        KernelPlugin plugin = KernelPluginFactory.CreateFromType<MenuPlugin>();
+        agent.Kernel.Plugins.Add(plugin);
+
+        // Create a chat for agent interaction. For more, see: Example03_Chat.
+        var chat = new TestChat();
+
+        // Respond to user input, invoking functions where appropriate.
+        await WriteAgentResponseAsync("Hello");
+        await WriteAgentResponseAsync("What is the special soup?");
+        await WriteAgentResponseAsync("What is the special drink?");
+        await WriteAgentResponseAsync("Thank you");
+
+        // Local function to invoke agent and display the conversation messages.
+        async Task WriteAgentResponseAsync(string input)
+        {
+            chat.AppendUserMessageToHistory(input);
+            this.WriteLine($"# {AuthorRole.User}: '{input}'");
+
+            await foreach (var content in chat.InvokeAsync(agent))
+            {
+                this.WriteLine($"# {content.Role} - {content.AuthorName ?? "*"}: '{content.Content}'");
+            }
+        }
+    }
+
+    public Example02_Plugins(ITestOutputHelper output)
+        : base(output)
+    {
+        // Nothing to do...
```

Comment:
I've seen this comment is multiple places, it doesn't add value so please remove it as others will copy this pattern